src/main/java/com/archyx/aureliumskills/support/PlaceholderSupport.java

```diff
@@ -261,6 +264,15 @@ else if (identifier.equals(skill.name().toLowerCase(Locale.ENGLISH) + "_roman"))
             }
         }
 
+        // Get Actionbar Status
+        if (identifier.startsWith("actionbar_status")) {
+            if (new ActionBar(AureliumAPI.getPlugin()).getPlayerActionbarEnableStatus(player)) {
```

Comment:
Creating a new ActionBar instance does not work because your instance will just have empty action bar data and will not be modified. You need to use `plugin.getActionBar()` to get the real instance that is actually used by the rest of the plugin.